docs/data/release_archive.yml

```diff
@@ -779,6 +779,9 @@ release_archive:
       release_date: 2017-07-27
 
   flink_cdc:
+    - version_short: 3.4
+      version_long: 3.4.0
+      release_date: 2025-04-26
```

Comment:
Release date should be updated when the release actually happens.

Reply:
Addressed.